cmd/k8s-bigip-ctlr/main.go

```diff
@@ -191,7 +192,9 @@ func _init() {
 	// TODO: Rephrase agent functionality
 	agent = bigIPFlags.String("agent", "cccl",
 		"Optional, when set to as3, orchestration agent will be AS3 instead of CCCL")
-
+	overrideAS3Decl = bigIPFlags.String("override-as3-declaration", "",
+		"Optional, when set, the JSON key/values from this ConfigMap will override"+
+			" key/values from internally generated AS3 declaration")
```

Comment:
We can improve this helper statement to be more informational. 

"Optional, provide Namespace where the ConfigMap is existing and name of that ConfigMap in format `<namespace>/<configmap-name>`. When set the JSON from ConfigMap will override internally generated AS3 declaration."

+ documentation update required.